seed/views/main.py

```diff
@@ -446,3 +450,159 @@ def delete_organization_inventory(request):
         'progress': 0,
         'progress_key': deleting_cache_key
     })
+
+
+def pm_integration(request):
+    return render(request, 'seed/pm_integration.html', {})
+
+
+ATTRIBUTES_TO_PROCESS = [
+    'national_median_site_energy_use',
+    'site_energy_use',
+    'source_energy_use',
+    'site_eui',
+    'source_eui'
+]
+
+
+def normalize_attribute(attribute_object):
```

Comment:
Need to decide what final units we want each of these to be and actually apply them properly.